.estafette.yaml

```diff
@@ -302,6 +327,24 @@ stages:
         container: dotnet
         versionTagSuffix: v8.0-sdk
 
+      check-efficiency-9.0:
+        image: extensions/docker:stable
+        action: dive
```

Comment:
https://github.com/estafette/estafette-extension-docker/blob/94ea9423dcb4bd7111e629b08c2433464558a490/main.go#L676

It seems "dive" and "trivy" already deprecated

Reply:
Thanks for the heads up, I removed those steps. (And `build` already has `severity` configured)